packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2371,27 +2436,36 @@ export class StripeHelper {
       const paymentMethod = customer.invoice_settings
         .default_payment_method as Stripe.PaymentMethod;
       if (paymentMethod.card) {
-        ({ last4: lastFour, brand: cardType } = paymentMethod.card);
+        ({ last4: lastFour, brand: cardType, country } = paymentMethod.card);
       }
+      if (paymentMethod.billing_details.address)
```

Comment:
Do we know `billing_details` is going to be defined? I.e. should we use the optional chain operator here?

Reply:
I don't remember seeing any warnings from the type checker while working on it.